fleet/lib/htcondor/overlays/manke/deployment-htcondor-worker.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: htcondor-worker
+  labels:
+    app: htcondor-worker
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: htcondor-worker
+  template:
+    metadata:
+      annotations:
+        k8s.v1.cni.cncf.io/networks: kube-system/htcondor
```

Comment:
Is the multus annotation still needed?  If not, can the `htcondor` metallb pools be removed?

Reply:
no, the Multus and MetalLB are still needed, I requested Julio to accommodate a small network just for HTCondor in LS and CP, so the networking part will be redone when the segments are ready, so I can keep HTCondor separate from the other services.

Reply:
Why does the worker need a multus interface when the `CONDOR_HOST` is internal to k8s?

There is no htcondor pool for ruka.

Reply:
the head node for manke and yagan are allocated inside other networks, but in ruka, everything happens inside the dev pool. (manke-yagan are waiting on new segments)